SoftLayer/CLI/virt/create.py

```diff
@@ -150,6 +153,13 @@ def _parse_create_args(client, args):
     return data
 
 
+def _add_keys(args, client, keys):
```

Comment:
I don't like functions that modify the data that gets passed in.  this function should `return keys` instead of modifying the array that gets passed in.